storybookjs/storybook
# Core: Add `globals` URL param and remove from sessionStorage

Issue: #11604

## What I did

Added `globals` URL query param that tracks any globals that have a value that deviates from their initial/default value.
Removed session storage for globals.

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? yes, todo

If your answer is yes to any of these, please make sure to include it in your PR.
